api/services/ocean/aws/paths/clusters-roll.yaml

```diff
@@ -10,7 +10,14 @@ post:
     - "Ocean AWS"
   parameters:
     - $ref: "../../../../commons/parameters/accountId.yaml"
-    - $ref: "../../commons/parameters/oceanId.yaml"
+    - in: "path"
```

Comment:
@vlad666spot  why changing?

Reply:
I experience issue with path param Ocean ID and made it inline - but ref is applicable so I change it back. Fixed